packages/gatsby-plugin-fastify/package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "gatsby-plugin-fastify",
+  "name": "gatsby-plugin-fastify-klyngen",
   "description": "Gatsby plugin for integration serving gatsby on Node.js using Fastify.",
-  "version": "0.9.0",
+  "version": "0.9.2",
```

Comment:
Version changes happen automatically, no need to bump. Please use the `yarn cs` command in the root to add a change set for these changes.

Reply:
Agreed :D